src/snowflake/cli/_plugins/nativeapp/same_account_install_method.py

```diff
@@ -53,8 +44,9 @@ def using_clause(
             return ""
 
         if self.version:
+            version_clause = f"version {to_identifier(self.version)}"
```

Comment:
Bug where numbers like `2` are passed in directly without being converted to identifier, leading to syntax errors